### ISSUE TITLE
failing to find services.json

### ISSUE THREAD

**Issue by nWidart** [orchestral/testbench#86]
Hello,

I'm getting an error when running an 'empty' test suite, with only a custom service provider loaded:

```
file_put_contents(/Users/nicolaswidart/Sites/Packages/Breezometer-laravel/vendor/orchestra/testbench/src/Traits/../../fixture/bootstrap/cache/services.json): failed to open stream: No such file or directory
```

Some permission problem maybe ?

Thank you,

**Comment by crynobone** [comment]
You testing against Laravel 5.1 already? :+1: 

I'll have this fixed soon.

**Comment by nWidart** [comment]
Hm on 5.0 still. 
Ah, I see, composer got testbench 3.1.

Thanks for the heads-up, forced it back to 3.0.* :)

**Comment by crynobone** [comment]
Fixed on 3.1 as well.

**Comment by nWidart** [comment]
Awesome :smiley:
